eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -331,4 +331,11 @@
     <Class name="com.azure.core.management.AzureProxy"/>
     <Bug pattern="NP_NULL_PARAM_DEREF"/>
   </Match>
+
+  <!--The public field already exists in the public API surface area-->
+  <Match>
+    <Class name="~com\.microsoft\.azure\.servicebus\.(TransactionContext|management\.ManagementClientConstants)"/>
+    <Field name="~(MAX_DURATION|NULL_TXN)"/>
+    <Bug pattern="MS_SHOULD_BE_FINAL"/>
+  </Match>
```

Comment:
[SpotBugs P1] 
NULL_TXN: This static field public but not final. 
MAX_DURATION isn't final but should be.